app/models/registration.rb

```diff
@@ -531,6 +534,34 @@ def serializable_hash(options = nil)
     super(DEFAULT_SERIALIZE_OPTIONS.merge(options || {}))
   end
 
+  def self.bulk_auto_accept(competition)
+    if competition.waiting_list.present?
+      competition.waiting_list.entries.each do |r_id|
```

Comment:
I think you can save that `if` by doing something like `competition.waiting_list&.entries&.each`, although I'm not sure whether that's easier to read or not